frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-subscribe.tsx

```diff
@@ -511,132 +508,16 @@ export const OperatorHubSubscribeForm: React.FC<OperatorHubSubscribeFormProps> =
-                </div>
-              </RadioInput>
-            </div>
-          </div>
-          <div className="form-group">
-            <h5 className="co-required">Installed Namespace</h5>
-            {selectedInstallMode === InstallModeType.InstallModeTypeAllNamespaces &&
-              globalNamespaceInstallMode}
-            {selectedInstallMode === InstallModeType.InstallModeTypeOwnNamespace &&
-              singleNamespaceInstallMode}
-          </div>
-          <div className="form-group">
-            <Tooltip content="The strategy to determine either manual or automatic updates.">
-              <h5 className="co-required">Approval Strategy</h5>
-            </Tooltip>
-            <RadioGroup
-              currentValue={selectedApproval}
-              items={[
-                { value: InstallPlanApproval.Automatic, title: 'Automatic' },
-                { value: InstallPlanApproval.Manual, title: 'Manual' },
-              ]}
-              onChange={(e) => setApproval(e.currentTarget.value)}
-            />
-          </div>
-        </>
-        <div className="co-form-section__separator" />
-        {formError()}
-        <ActionGroup className="pf-c-form">
-          <Button onClick={() => submit()} isDisabled={formValid()} variant="primary">
-            Install
-          </Button>
-          <Button variant="secondary" onClick={() => history.push('/operatorhub')}>
-            Cancel
-          </Button>
-        </ActionGroup>
-      </div>
-      <div className="col-xs-6">
-        <ClusterServiceVersionLogo
-          displayName={_.get(channels, '[0].currentCSVDesc.displayName')}
-          icon={iconFor(props.packageManifest.data[0])}
-          provider={provider}
-        />
-        <h4>Provided APIs</h4>
-        <div className="co-crd-card-row">
-          {!providedAPIs.length ? (
-            <span className="text-muted">No Kubernetes APIs are provided by this Operator.</span>
-          ) : (
-            providedAPIs.map((api) => (
-              <CRDCard key={referenceForProvidedAPI(api)} canCreate={false} crd={api} csv={null} />
-            ))
-          )}
-        </div>
-      </div>
-    </div>
-  );
-};
-
-const OperatorHubSubscribe: React.FC<OperatorHubSubscribeFormProps> = (props) => (
-  <StatusBox data={props.packageManifest.data[0]} loaded={props.loaded} loadError={props.loadError}>
-    <OperatorHubSubscribeForm {...props} />
-  </StatusBox>
-);
-
-export const OperatorHubSubscribePage: React.SFC<OperatorHubSubscribePageProps> = (props) => {
-  const search = new URLSearchParams({
-    'details-item': `${new URLSearchParams(window.location.search).get(
-      'pkg',
-    )}-${new URLSearchParams(window.location.search).get('catalogNamespace')}`,
-  });
+  if (showInstallStatusPage) {
+    return (
+      <OperatorInstallPage
+        targetNamespace={selectedTargetNamespace}
+        activeNamespace={new URLSearchParams(window.location.search).get('targetNamespace') || null}
```

Comment:
I would get the active namespace from Redux inside the `OperatorInstallPage` component instead of passing it as a prop.